docs/jetting.md

```diff
@@ -0,0 +1,709 @@
+# How to write a jet
+
+A guide for new Urbit developers.
+
+Problem statement: coder A has written some hoon, and you - coder B - need to jet it.
+
+Preconditions:
+
+* You have a pill `jetted.pill`, located in ~/tlon.
+* You have an example of hoon code that, executed in dojo, does the right thing.
+
+## Get all the source code you need.
+
+The source code you will need to modify lives in two different repos:
+urbit and arvo.  The urbit repo has the source code for the
+intrepretter / personal server, written in C.  The arvo repo has the
+operating system, written in hoon.  Since you're jetting the hoon, you
+need to modify both.
```

Comment:
This is a nitpick, but you should use "Hoon" (capitalized) rather than "hoon" when you're referring to the programming language.  A "hoon" is a (certain kind of) Hoon expression.  Not making this distinction can be slightly confusing in places.